client/src/assets/scss/overrides.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Vuetify Overrides
+ */
+
+@import "theme.scss";
+
+.v-application {
+  font-size: 1rem; /* 16px */
+  line-height: 1.5rem; /* 24px */
+  font-family: 'BCSans', Verdana, Arial, sans-serif !important;
+  padding: 0;
+}
+
+.v-application .text-body-1,
+.v-application .text-body-2 {
+  font-family: 'BCSans', Verdana, Arial, sans-serif !important;
+}
+
+.v-btn {
+  letter-spacing: normal;
+  border-bottom-width: 1px !important;
```

Comment:
How will this conflict with the existing sass overrides for `v-btn`?

Reply:
Empirically, I can say that both SCSS and SASS are in play, so yes, this affects existing buttons (and this is a style update from Tracey).

Since the SCSS files are imported after the SASS files, this code overrides SASS styles.

Reply:
I wonder if it retains the properties defined earlier that are not re-defined here. For example `font-size` and `text-transform` etc etc

Reply:
My assumption, is those properties will still be retained. Either way, nothing major!

Reply:
Yes, this code only overrides some properties. The other properties are retained.